src/ecc/rtl/ecc_reg.rdl

```diff
@@ -307,13 +291,12 @@ addrmap ecc_reg {
 
     /* ---- ECC Component Input IV ---- */
     reg {
-        name = "ECC component IV register type definition";
-        desc = "12 32-bit registers storing the 384-bit IV required 
+        name = "ECC IV";
+        desc = "ECC component IV register type definition
+                12 32-bit registers storing the 384-bit IV required 
                 for SCA countermeasures to randomize the inputs with no change
                 on the ECC outputs. 
-                The IV can be any 384-bit value in [0 : 2^384-1].
-                These registers are located at ECC_base_address + 
-                0x0000_0480 to 0x0000_04AC in big-endian representation.";
+                The IV can be any 384-bit value in [0 : 2^384-1].";
```

Comment:
big-endian

Reply:
no need for IV since it's a random value